app/src/main/sentences/hi/lyrics.dslf

```diff
@@ -0,0 +1,5 @@
+lyrics: high
+get|show|load|display|ढूंढें|(खोजें?)? मुझे? ? Lyrics|पाठ (for|of|from|about (गीत)?)? ।गीत।;
+गाना गाएँ)? ।गीत।;
+।गीत। गीत|पाठ;
+गाओ|क्या|(क्या है|एस) वो|गाना चल रहा है|(वो|जो जाता है|कहता है|बताता है|गाता है)|(गीत के साथ) .गाना.;
```

Comment:
This looks a bit strange to me. The words `है` and `वो` are repeated multiple times. Does the following sentence make sense in Hindi? "गाओ वो चल रहा वो जाता है है है है ..."? It should roughly mean "What's the song that goes ...", but when using a translator I get "sing he's walking he goes he's he's", which doesn't seem right. But that sentence perfectly matches the pattern you wrote, so I think it should be changed.